cmd/service.go

```diff
@@ -231,6 +234,27 @@ func runServiceStatusCmd(cmd *cobra.Command, args []string) {
 
 	if !db.IsInstalled() {
 		fmt.Println("Steampipe database service is NOT installed")
+	} else if viper.GetBool(constants.ArgAll) {
+		var processes []*psutils.Process
+		var err error
+
+		gotDetails := make(chan bool)
+		sp := display.StartSpinnerAfterDelay("Getting details", constants.SpinnerShowTimeout, gotDetails)
+		defer func() {
+			close(gotDetails)
+			display.StopSpinner(sp)
+			printAllStatus(processes)
```

Comment:
this code feels a little unclear, do we really need to put `printAllStatus` in the defer?

Reply:
refactored and fixed